credits.txt

```diff
@@ -309,6 +328,7 @@ contributed to Endless Sky:
   wjp
   Wrzlprnft
   YellowApple
+  yjhn
```

Comment:
yjhn is tatami4, who is already in the credits. Maybe remove the old name as well so we can use the new one.

Reply:
Do we know if they prefer the new one or the old one, or if replacing the old one with the new one could cause safety issues?

Just comes to mind that if someone has stopped contributing as X and is now using account Y, it may be because, for safety, they've had to stop using X name. Suddenly having X name replaced by Y could be a giveaway to someone watching that this name change has taken place. Even if we (inside the community) know that X=Y, If there's any chance of there being a safety issue, I'd rather just have both names listed.

That being said, if they just changed their github account name... then that's a non-issue.